### PR TITLE
chore: release 0.1.0 

### PR DESCRIPTION
**Pull request by wrn14897**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -15,32 +15,30 @@ To instrument your React Native application, follow these steps.
...
 3. Customize the initialization parameters to specify:
 
-- `realm`: The Splunk Observability Cloud realm of your organization. For example, `us0`.
-- `rumAccessToken`: Your Splunk RUM authentication token. You can find or generate the token [here](https://app.signalfx.com/o11y/#/organization/current?selectedKeyValue=sf_section:accesstokens). Notice that RUM and APM authentication tokens are different.
-- `applicationName`: Name of your application. Set it to distinguish your app from others in Splunk Observability Cloud.
+- `apiKey`: Your HyperDX API key. You can find it [here](https://www.hyperdx.io/team).
+- `service`: Name of your application. Set it to distinguish your app from others in HyperDX.
 
-> If needed, you can set a different target URL by specifying a value for `beaconEndpoint`. Setting a different beacon URL overrides the `realm` setting.
+> If needed, you can set a different target URL by specifying a value for `beaconEndpoint`.
```
Comment (MikeShi42) [inline README.md]: Thoughts on aligning this with the browser package and calling it `url`? Don't have strong feelings either way

File: README.md [hunk 2]
```diff
@@ -15,32 +15,30 @@ To instrument your React Native application, follow these steps.
...
 });
 
 ```
 
 3. Customize the initialization parameters to specify:
 
-- `realm`: The Splunk Observability Cloud realm of your organization. For example, `us0`.
-- `rumAccessToken`: Your Splunk RUM authentication token. You can find or generate the token [here](https://app.signalfx.com/o11y/#/organization/current?selectedKeyValue=sf_section:accesstokens). Notice that RUM and APM authentication tokens are different.
-- `applicationName`: Name of your application. Set it to distinguish your app from others in Splunk Observability Cloud.
+- `apiKey`: Your HyperDX API key. You can find it [here](https://www.hyperdx.io/team).
```
Comment (MikeShi42) [inline README.md]:
```suggestion
- `apiKey`: Your HyperDX Ingestion API key. You can find it [here](https://www.hyperdx.io/team).
```

File: android/src/main/java/com/splunkotelreactnative/crash/CrashReporterAttributes.java [hunk 3]
```diff
@@ -6,9 +6,9 @@
 
 import io.opentelemetry.api.common.AttributeKey;
 
-// copied from splunk-otel-android SplunkRum class
+// copied from splunk-otel-android HyperDXRum class
```
Comment (MikeShi42) [inline android/src/main/java/com/splunkotelreactnative/crash/CrashReporterAttributes.java]: Probably an erroneous find/replace here

File: README.md [hunk 4]
```diff
@@ -126,20 +124,3 @@ Supported features:
 - Capturing native crashes
 
 For more information about how this library uses Opentelemetry and about future plans check [CONTRIBUTING.md](CONTRIBUTING.md#Opentelemetry).
-
-## License
```
Comment (MikeShi42) [inline README.md:130]: We can probably keep a link here pointing to the upstream Splunk repo

File: src/globalAttributeAppender.ts [hunk 5]
```diff
@@ -26,7 +26,7 @@ export default class GlobalAttributeAppender implements SpanProcessor {
 
   onStart(span: Span): void {
     span.setAttributes(getGlobalAttributes());
-    span.setAttribute('_splunk_operation', span.name);
+    span.setAttribute('_hyperdx_operation', span.name);
```
Comment (MikeShi42) [inline src/globalAttributeAppender.ts:29]: Do you know what this is used for? Not immediately clear to me what will we do with this property (unless span_name isn't set properly for some reason?)

**Review by MikeShi42 — APPROVED** [review]

File: README.md (hunk 1 above)
Comment (wrn14897) [reply README.md]: Sure, why not. But I'd prefer to do this in the next iteration

File: src/globalAttributeAppender.ts (hunk 5 above)
Comment (wrn14897) [reply src/globalAttributeAppender.ts:29]: Not very clear. I saw this appeared in the android custom zipkin encoder. we might be able to remove it but I'd prefer to do it in the next release

**Pull request closed without merging**